src/SFA.DAS.ApprenticeCommitments.Web/Pages/Apprenticeships/TransactionComplete.cshtml

```diff
@@ -9,18 +9,22 @@
 
             <div class="govuk-panel govuk-panel--confirmation">
                 <h1 class="govuk-panel__title">
-                    You have confirmed your apprenticeship
+                    You have confirmed your apprenticeship details.
                 </h1>
                 <div class="govuk-panel__body">
                     @Model.CourseName
                 </div>
             </div>
 
-            <p>We have sent you a confirmation email.</p>
-
             <h3 class="govuk-heading-m">What happens next?</h3>
 
-            <p>We will send the confirmation of your apprenticeship to your employer and training provider and they will be in touch to start your apprenticeship.</p>
+            <p>You have completed the confirmation of your apprenticeship details. Your employer and training provider will contact you.</p>
+
+            <h3 class="govuk-heading-m">Can you help?</h3>
+
+            <p>This is a new service and we want to hear what you think. Please complete this 5 minute survey. Your feedback will help us make improvements to the service.</p>
+
+            <a class="govuk-button" data-module="govuk-button" href="https://forms.office.com/Pages/ResponsePage.aspx?id=yXfS-grGoU2187O4s0qC-XVnVtImmRBOiQ5Eo7TmxhBURDVJOFZSSDNIM05JUlpTWkwzWEVXOEE5Ny4u" target="_blank">Take the survey</a>
```

Comment:
Recommended to add `noopener noreferrer` ([e.g.](https://github.com/SkillsFundingAgency/das-apprentice-portal-web/blob/a01d05d0891b5000a7f945e6b9efe6c6e01cba2a/src/SFA.DAS.ApprenticePortal.SharedUi/Views/Shared/_BetaBanner.cshtml#L11))
```suggestion
            <a class="govuk-button" data-module="govuk-button" rel="noopener noreferrer" href="https://forms.office.com/Pages/ResponsePage.aspx?id=yXfS-grGoU2187O4s0qC-XVnVtImmRBOiQ5Eo7TmxhBURDVJOFZSSDNIM05JUlpTWkwzWEVXOEE5Ny4u" target="_blank">Take the survey</a>
```

Reply:
Done